lib/api/writeFile.js

```diff
@@ -78,7 +78,7 @@ module.exports = function writeFile(filename, data, options, cb) {
     function callback() {
       return function(err) {
         if (stop) return;
-        if (err) {
+        if (err && err.code !== 'STATUS_PENDING') {
```

Comment:
I don't understand this change, what does this error mean and what should we do?

Should we resend this packet?

If not, how can we wait for this packet to be written?

Reply:
AFAICT, this error only happens in case of async requests/responses which are not currently used in this library.

Reply:
but ```smb2Client.createWriteStream``` is asynchronous, and I got this error.